core/src/main/java/io/kestra/core/models/flows/input/FileInput.java

```diff
@@ -22,10 +23,61 @@ public class FileInput extends Input<URI> {
 
     @Deprecated(since = "0.24", forRemoval = true)
     public String extension;
+    
+    /**
+     * The allowed file extensions or MIME types.
+     * Example: ".csv,.txt" or "text/csv,text/plain"
+     * Can contain either file extensions starting with dots (.pdf, .txt) or MIME types (text/plain)
+     */
+    private String accept;
```

Comment:
```suggestion
    private List<String> acceptedExtensions;
```

It's clearer like that.
I don't see the point of supporting both extension and mime-type.
Mime-type are usually used in HTTP where you don't have extensions, for files, it's usually accepted that the type of the file is defined by its extension so I think supporting extension validation is enought here.